support-models/src/main/scala/com/gu/support/workers/AcquisitionData.scala

```diff
@@ -1,19 +0,0 @@
-package com.gu.support.workers
```

Comment:
moved to support-models/src/main/scala/com/gu/support/acquisitions/AcquisitionData.scala